migration.py

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/python
+
+import os
+import argparse
+import sys
+import shutil
+
+HASURA_PATH = "./deployment/hasura/"
+MIGRATION_PATH = HASURA_PATH+"migrations/"
+
+# Create a cli parser
+my_parser = argparse.ArgumentParser(description='Migrate an AERIE Database')
```

Comment:
Feel free to take this or leave it but lately I've had my `ArgumentParser` description point directly to the top-level pydoc string.

To do this just change the script's preamble to:
```py
#!/usr/bin/env python3
'Migrate an AERIE Databse'
import os
...
```
Then you can write:
```py
my_parser = argparse.ArgumentParser(description=__doc__)
```

Reply:
Thanks, Paul, relatively new to python so I can take all the suggestions.